### ISSUE TITLE
Add 1s data to DispatchPrice

### ISSUE THREAD

**Issue by Starstrike99** [UNSW-CEEM/NEMOSIS#44]
Could you please add the new columns "RAISE1SECRRP" and "LOWER1SECRRP" to DispatchPrice, to allow download of 1s FCAS prices?

**Comment by nick-gorman** [comment]
Good idea. Until I get around to it there is also this workaround as an option:
https://github.com/UNSW-CEEM/NEMOSIS#accessing-additional-table-columns
